PhotonTheme.Shared/Converters/StringToVisibility.cs

```diff
@@ -0,0 +1,40 @@
+﻿// // --------------------------------------------------------------------------------------------------------------------
```

Comment:
This copyright does not look beautiful, I believe that it can be made better.

Reply:
The copyright has been a mistake on my part due to Resharper auto-generator. I don't want any copyright in any file and thank you for noticing!